### PR TITLE
Dont error on unknown block types

### LINKED ISSUE #579: Failure trying to read block for Zoom call message
Error "unsupported block type" received trying to parse this message in conversations.history.

        {
            "type": "message",
            "subtype": "bot_message",
            "text": "smith has started a meeting",
            "ts": "1566248229.004500",
            "bot_id": "ABCDEFGHI",
            "blocks": [
                {
                    "type": "call",
                    "call_id": "ABCDEFGHI",
                    "block_id": "jHZ6",
                    "api_decoration_available": false,
                    "call": {
                        "v1": {
                            "id": "ABCDEFGHI",
                            "app_id": "ABCDEFGHI",
                            "app_icon_urls": {
                                "image_32": "some-url.png",
                                "image_36": "some-url.png",
                                "image_48": "some-url.png",
                                "image_64": "some-url.png",
                                "image_72": "some-url.png",
                                "image_96": "some-url.png",
                                "image_128": "some-url.png",
                                "image_192": "some-url.png",
                                "image_512": "some-url.png",
                                "image_1024": "some-url.png",
                                "image_original": "some-url.png"
                            },
                            "date_start": 1566248229,
                            "active_participants": [],
                            "all_participants": [
                                {
                                    "slack_id": "ABCDEFGHI"
                                },
                                {
                                    "external_id": "12345678",
                                    "avatar_url": "",
                                    "display_name": "John Smith"
                                }
                            ],
                            "display_id": "123-456-789",
                            "join_url": "https:\/\/blah.zoom.us\/j\/123456789",
                            "name": "Zoom meeting started by smith",
                            "created_by": "ABCDEFGHI",
                            "date_end": 1566249369,
                            "channels": [
                                "ABCDEFGHI"
                            ],
                            "is_dm_call": false,
                            "was_rejected": false,
                            "was_missed": false,
                            "was_accepted": false,
                            "has_ended": true
                        }
                    }
                }
            ]
        },

### PR DESCRIPTION
**Pull request by acaloiaro**
Fixes #579 

##### Pull Request Guidelines

These are recommendations for pull requests.  
They are strictly guidelines to help manage expectations.

##### should this be an issue instead
- [ ] is it a convenience method? (no new functionality, streamlines some use case)
- [ ] exposes a previously private type, const, method, etc.
- [ ] is it application specific (caching, retry logic, rate limiting, etc)
- [ ] is it performance related.

##### API changes

Since API changes have to be maintained they undergo a more detailed review and are more likely to require changes.

- no tests, if you're adding to the API include at least a single test of the happy case.
- If you can accomplish your goal without changing the API, then do so.
- dependency changes. updates are okay. adding/removing need justification.

###### examples of API changes that do not meet guidelines:
- in library cache for users. caches are use case specific.
- Convenience methods for Sending Messages, update, post, ephemeral, etc. consider opening an issue instead.

### CODE REVIEW

File: errors.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package slack
 
-import "github.com/nlopes/slack/internal/errorsx"
+import "github.com/acaloiaro/slack/internal/errorsx"
```
Comment (gregmarr) [inline errors.go]: @acaloiaro I think the PR is failing to build in Travis because of these import statement changes.

Comment (acaloiaro) [reply errors.go]: Ah good catch; thanks. I never checked in on this after PRing. Will fix when I have the time.

Comment (acaloiaro) [reply errors.go]: Thanks @gregmarr -- tests are passing now.

Comment (gregmarr) [reply errors.go]: You're welcome, and thanks for the fix.

**Comment by ollieparsley** [comment]
@acaloiaro thanks for your contribution. The UnknownBlock change was also implemented in #669, which has already been merged. I'll close this PR.

**Comment by acaloiaro** [comment]
Great @ollieparsley. I'm happy someone has taken over this project again. Cheers.

**Pull request closed without merging**